iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergTableUtil.java

```diff
@@ -559,4 +566,32 @@ public static List<String> getPartitionNames(Table icebergTable, Map<String, Str
       throw new SemanticException(String.format("Error while fetching the partitions due to: %s", e));
     }
   }
+
+  public static Snapshot getSpecificSnapshot(org.apache.hadoop.hive.ql.metadata.Table hmsTable, Table table) {
```

Comment:
could you please rename to `getTableSnapshot`

Reply:
Done